app/views/changes/html.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= stylesheet_link_tag "changes", media: "all" %>
+
+<div class="klax-section klax-topper">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h1>Latest Captured HTML</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-6">
+        <p class="lead">Below is a version of <b><%= @page&.name || @page&.url %></b> downloaded on <%= @timestamp.strftime("%A, %B %d, %Y at %l:%M %P %z off UTC") %>. </p>
+        <p class="lead">It may take a moment for the captured page to load. The page may look odd because we're not capturing the CSS as well. Sorry.</p>
+      </div>
+
+      <div class="col-md-3">
+        <!-- negative space -->
+      </div>
+      <div class="col-md-3">
+       <%= link_to 'Past Snapshots', snapshots_page_path(@page), class: 'btn btn-default btn-lg btn-block' %>
+        <%= link_to 'Download HTML', download_html_path(@change_id), download:true, class: 'btn btn-default btn-lg btn-block' %>
+        <%= link_to 'Go to Source', @page.url, class: 'btn btn-default btn-lg btn-block' %>
+        <%= link_to 'Edit Klaxon', edit_page_path(@page), class: 'btn btn-default btn-lg btn-block' %>
+
+      
+      </div>
+    </div>
+  </div></div>
+<div class="klax-section klax-spacer-lg">
+  <div class="container klax-spacer-md">
+    <div class="row klax-spacer-md">
+
+<div class="col-md-12" style="border:solid;"><iframe src="/changes/page/<%= @change_id %>/html/raw"></div> </div>
+      </div>
+
+    </div></div></div>
+    </div>
+  </div><!-- /.container -->
+</div>
```

Comment:
Some funky indentation here that makes it hard to parse the HTML structure. Make sure each `<div> </div>` pair is either on the same line as its counterpart or each is on its own line, then reindent the file.

Reply:
Oh yeah. Gotcha.